teslausb-www/html/filebrowser.html

```diff
@@ -37,6 +37,7 @@
 
 var fb = document.querySelector(".filebrowser");
 new FileBrowser(fb, "/var/www/html/Music", "Music", "Music");
+new FileBrowser(fb, "/var/www/html/LightShow", "LightShow", "LightShow");
```

Comment:
This won't work, as it's attaching both FileBrowsers to the same div in the html document.
It doesn't really matter though since this html file is not used.